.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug Report
+about: Help us fix something that's broken.
+title: "PackageName:  General Summary of the Bug"
+labels: "type: bug"
+assignees: ''
+
+---
+
+## Bug Report
+@trilinos/PackageName
+
+### Description
+What went wrong?  What should have happened?  Do you have an idea what might
```

Comment:
I don't think we need any text here.  Most readers will understand what "Description" means.  Also, the first thing most people do when they fill out an issue is delete most of the template -- it's better if we keep it as short as possible, so that readers don't delete something they should fill out.

Reply:
It seems we should be giving some guidance, otherwise there's not much point in having the template.  Particularly I find people often leave out the broader context of what they're trying to do.

Reply:
I don't mean to block your work -- it's just that every time we open an issue, we delete almost everything that we see.  Maybe we're not the target audience, though ;-)

Reply:
No worries.  I think since this is already a significant reduction in the template text, we'll probably leave things as is, unless someone on the framework team wants to chime in.